yarn.lock

```diff
@@ -2,34 +2,59 @@
 # yarn lockfile v1
 
 
-"@sinonjs/formatio@^2.0.0":
-  version "2.0.0"
-  resolved "http://registry.npmjs.org/@sinonjs/formatio/-/formatio-2.0.0.tgz#84db7e9eb5531df18a8c5e0bfb6e449e55e654b2"
+"@sinonjs/commons@^1.0.2":
```

Comment:
What are these changes needed for?

Reply:
If you just run a plain `yarn` (or `yarn install`), it adds these in. They're dev dependencies anyway so I guess it doesn't matter too much?

Reply:
I think we should exclude it from this PR if we don't need it then. Might end up being a breaking change depending on people's yarn/node versions